hummedia/api.py

```diff
@@ -35,6 +35,17 @@ def Collection(collection,id=None):
 	else:
 		return endpoint_404()
 
+@app.route('/video/stale', methods=['GET', 'OPTIONS'])
+@app.route('/video/stale/<date>', methods=['GET', 'OPTIONS'])
+@crossdomain(origin=CROSS_DOMAIN_HOSTS,headers=['Origin','x-requested-with','accept','Content-Type'],credentials=True)
+def getStaleMedia(date=None):
+    return MediaAsset(request).get_stale(date)
+
+@app.route('/video/<id>/view', methods=['GET', 'PUT', 'OPTIONS'])
```

Comment:
If the `PUT` action won't be used, it can probably just be removed here. Doesn't make much sense for something that isn't idempotent, anyway.